components/badges/badge.css

```diff
@@ -0,0 +1,92 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700;800&family=Roboto:ital,wght@0,300;0,400;0,500;0,700;0,900;1,100&display=swap');
+
+
+
+.badge-container{
+position: relative;
+}
+
+.online-img-badge{
+    background-color:rgb(38, 207, 38);
+    border-radius:50%;
```

Comment:
Hey ! I would suggest you to have a separate **.badge class** where you can add badge related properties and you can only change the color of the badge according to the necessity. 
Eg : .online {
             color : yellow;
       }
There are few properties repeated in each badge.

Reply:
Well the color is specific to the badge type like online is green only offline is red only that is why

Reply:
I meant the border-radius , border , position which are repetitive. 

Okay , keep going